lib/builder/components/SavedQueriesSelect.js

```diff
@@ -56,7 +56,9 @@ class SavedQueriesSelect extends Component {
     }
   }
 
-  selectSavedQuery = (e, index) => {
+  selectSavedQuery = (e, index) => {console.log(this.props.value)
+    const { value: oldValue } = this.props;
+    this.props.changeSavedQueryList(oldValue, e);
```

Comment:
Based on `changeSavedQueryList` action definition - the function accepts old and new value. Why in second argument the full event object is passed ?

In reducer the value is extracted from `event` - for reference `let delIndex = savedQueriesList.indexOf(el.value);`

Could we please, extract the value from `event` at container level and simplify reducer to not operate on JavaScript events ?